src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -192,7 +192,7 @@ public async Task<ImmutableArray<CodeFixCollection>> GetFixesAsync(Document docu
             if (result.Count > 0)
             {
                 // sort the result to the order defined by the fixers
-                var priorityMap = _fixerPriorityMap[document.Project.Language].Value;
+                ImmutableDictionary<CodeFixProvider, int> priorityMap = _fixerPriorityMap[document.Project.Language].Value;
```

Comment:
?

Reply:
Explained above: https://github.com/dotnet/roslyn/pull/40755#discussion_r364506292

In short, the compiler still has some issue with analyzing local functions. The above code fails to bootstrap if the type is annotated with `?` (or is `var` which is the same).